modules/flash-script.nix

```diff
@@ -124,6 +124,17 @@ in
             type = types.listOf types.str;
             default = [];
           };
+
+          taPublicKey = mkOption {
```

Comment:
Could you rename this to `taPublicKeyFile` for consistency with the other options?

Reply:
done!